packages/react-native-win32/src/Libraries/Components/View/ViewWin32.Props.ts

```diff
@@ -97,6 +97,7 @@ export type AccessibilityAnnotationInfo = Readonly<{
   typeName?: string;
   author?: string;
   dateTime?: string;
+  target?: string;
```

Comment:
On the UIA side IIRC this would need to map to an element in the UIA tree (in other words, an object that implements IRawElementProviderSimple). What would this string property map to on the JavaScript side?

Reply:
I have a corresponding native change that allows clients to register/retrieve a reference to a UIA object using a unique string ID. The target prop here uses that same ID so the native side can look up the previously registered object and use it as the target prop.

Reply:
Thanks @ksiler . I'd love to understand the design a bit more. If both the registration and the retrieval are done on the native side, why does the string ID need to be exposed on the JS side? Could you share a link to the native change? (Let's follow up offline if the code is in the internal Office codebase).